### PR TITLE
Update README.rst

### PR DESCRIPTION
**Pull request by EthanIDR**
added area for docker auth

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -73,6 +73,19 @@ the converted output to:
...
 `API`_
 
+--------------
+
+Changes for docker version
+--------------------------
+
+For our docker version authorization can be set to adjust the client to use authorization, 
+to do this simply ad a tuple containing the username an password to the initialization for BuildVu.
+::
```
Comment (elepedus) [inline README.rst]: This seems a bit wooly to me. Can we re-word to something like:

"If your BuildVu service requires authentication, you can set the username and password by passing an additional tuple argument like this:"

Comment (EthanIDR) [reply README.rst]: Looks good

**Comment by EthanIDR** [comment]
@KieranFrance, any thoughts?

**Comment by EthanIDR** [comment]
also will update of the README require an update of version number?

**Comment by KieranFrance** [comment]
Looks good to me, if the update is only to the read me file the version number should not need an update

**Comment by leonatherton** [comment]
It might still be worth incrementing the version so that the new documentation can be pushed to the package manager.

I'm not sure about pypi, but I know on NPM it stores a copy of the readme rather then using latest from the repository.

**Comment by KieranFrance** [comment]
Good call

**Pull request merged**